functional_tests/basic_java_test/tests.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2016 Jaedyn K. Draper
```

Comment:
In a complete contradiction with a comment I just made... I feel like we should change these to Copyright Sleeping Cat, LLC